addons/account/models/account.py

```diff
@@ -944,7 +944,7 @@ class AccountReconcileModel(models.Model):
         ('percentage', 'Percentage of balance')
         ], required=True, default='percentage')
     amount = fields.Float(digits=0, required=True, default=100.0, help="Fixed amount will count as a debit if it is negative, as a credit if it is positive.")
-    tax_id = fields.Many2one('account.tax', string='Tax', ondelete='restrict', domain=[('type_tax_use', '=', 'purchase')])
+    tax_ids = fields.Many2many('account.tax', string='Taxes', ondelete='restrict', domain=[('type_tax_use', '=', 'purchase')])
```

Comment:
Data types can not change in stable.

BTW, with the new accounting all the logic requires an account move line per tax with that is enought.

Reply:
Thanks for this hint, am I correct to assume that you refer to this method https://github.com/odoo/odoo/blob/11.0/addons/account/models/account_move.py#L370? 

In such case, it still needs `line.tax_ids` to work and I guess it's main purpose is speeding up / simplifying reporting or to circumvent the problem with 2many domain leafs which detail's I don't have at hand atm...

But I might have misunderstood...

Reply:
It should be cherry-picked to master, obviously, as I don't think the editor will choose to include in stable and provide migration scripts, although those would not be difficult...

Reply:
> Thanks for this hint, am I correct to assume that you refer to this method https://github.com/odoo/odoo/blob/11.0/addons/account/models/account_move.py#L370?

Yes.

As a side comment:

tax_ids was a `mess` always, the point is that once you do not know which is the base amount for a single tax then you will not have a unique pair relation between tax and amounts on the line.

I prefer for the sake of simplicity the "Convention against Configuration", then if you follow the convention of 1 tax 1 move line your live will be lovelly (at the en a m2m relation does that jost that in another table and making more complex the coding and the inverse computation.

Regards.

Reply:
It was! :smile: And I do agree wholehartedly. Yes, it happens :wink:
We should anyway make peace on our past controversies, as you can consider to start to appreciate my inputs.. :smile: